code/index.html

```diff
@@ -3,20 +3,117 @@
   <head>
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <!--external CSS connection-->
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <!-- meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+<!--menu-->
+<header class="mobilemenu">
+	<div class="dropdownlogo" style="float:left;">
+      	<img class="item1" src="img/logo.jpg" alt="logo"/>
+      	<div class="item2">Visit St.Martin</div>
+	</div>
+	<div class="dropdown" style="float:right;">
+  		  <button class="dropbtn"><i class="fa fa-bars"></i></button>
+   <div class="dropdown-content">
+        <div class="item3"><a href="#home">Home</a></div>
+        <div class="item4"><a href="#hotel">List of Hotel</a></div>
+        <div class="item5"><a href="#blog">Blog</a></div>
+        <div class="item6"><a href="#contact">Contact</a></div>
+  </div>
+</div>
+</header>
 
+<!--Main text upper part-->
     <section class="big-news">
-      <!-- The big news content -->
+      <div class="polaroid">
+        <img src="img/travel5.jpg" alt="coconut tree"/>
+        <p>Coconut tree</p>
+      </div>
+      <div class="big-news-text">
+        <h1>Natural piece of paradise</h1>
+        <p>
+          The St. Martin island and the journey to and fro, 
+          both are extremely enjoyable be it with family or friends or couple. 
+          It takes a good 2 to 2.5 hrs drive from Cox bazaar to Teknaf from 
+          where one has to board ferry to reach the island. The boat ride itself is really 
+          enjoyable where you’ll be accompanied by seagulls through and through. 
+          The travellers throw tit-bits and bread pieces for which the gulls throng 
+          along the boat till great distances <a href="https://travelvibe.net/saint-martin-the-ultimate-travelers-guide/" target="_blank"><i>read more</i></a>
+        </p>
+      </div>
     </section>
+<!--Main text lower part-->
+    <section class="mainText">
+      <div class="childOne">
+        <div class="polaroid">
+          <img src="img/coral island.jpg" alt="Coral island"/>
```

Comment:
Same comment here for the `img`-tag as above. I guess it's a matter of what you prefer also, but maybe it can make it more easy to read? :)